modules/core/src/v2/coins/eth.ts

```diff
@@ -36,6 +36,7 @@ import { Util } from '../internal/util';
 import { EthereumLibraryUnavailableError } from '../../errors';
 import { BaseCoin as StaticsBaseCoin, EthereumNetwork } from '@bitgo/statics';
 import { checkKrsProvider, getIsKrsRecovery, getIsUnsignedSweep } from '../recovery/initiate';
+import type * as EthTxLib from '@ethereumjs/tx';
```

Comment:
this should get erased at compile time, so even if this is an optional dependency should be fine.